core/lib/instrument_api.h

```diff
@@ -6219,6 +6219,8 @@ DR_API
  * have been flushed. The context to use can be obtained via
  * dr_get_mcontext() with the exception of the pc to continue at which must be passed as
  * an argument to the callout (see instr_get_app_pc()) or otherwise determined.
+ * For 32-bit ARM, be sure to use dr_app_pc_as_jump_target() to properly set the ISA
```

Comment:
This comment should probably be at `dr_redirect_execution` too, as the continuation pc is passed to that method.

I see other example usages where this is not done currently. Eg https://github.com/DynamoRIO/dynamorio/blob/c1be309c8f21f9eabd05b759378ee097e1af83b0/api/samples/cbr.c#L261

Reply:
dr_redirect_execution already has a comment about this, but perhaps it could be reworded.

Reply:
Moved the doc to dr_redirect_execution, merged with what was already there, and raised it up rather than being a lower-down note.

Updated a sample, bbdup, and several tests.